### PR TITLE
add-PST-classes

### PR DESCRIPTION
**Pull request by Kgomotso196**
# PROJECT TO DOs
### IN THIS PROJECT I:
1. Implemented classes and objects in Ruby.
2. Implemented encapsulation and inheritance with Ruby.
3. Ran a program using the command line.
4. Added the `Person` class that includes instance variables for `@id`, `@name`, and `@age`, with optional default values. It features methods to check age eligibility and permission for using services.
5. Added the `Student` class that inherits from `Person`, adding a constructor with `@classroom` and a `play_hooky` method returning `"¯\(ツ)/¯"`.
6. Added the `Teacher` class that inherits from `Person`, extending the constructor with `@specialization` and overriding `can_use_services?` to always return true.
7. Organized each class into its own separate file.
8. Made sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
9. Made sure that I used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
10. Made sure that I documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [Setup](#setup)
+- [💻 Getting Started](#getting-started)
   - [Prerequisites](#prerequisites)
-  - [Usage](#usage)
+  - [Setup](#setup)
+  - [Run tests](#run-tests)
```
Comment (TedLivist) [inline README.md:16]: - [ ] This run tests link does not work. Kindly fix it or remove it.

File: student.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require_relative '/person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, _classroom, name = 'unknown', parent_permission: true)
+    super(age, name, parent_permission)
```
Comment (TedLivist) [inline student.rb]: - [ ] Kindly assign `parent_permission` correctly.

  ```
  super(age, name, parent_permission: parent_permission)
  ```

File: teacher.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+require_relative '/person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization, age, name = 'unknown', parent_permission: true)
+    super(age, name, parent_permission)
```
Comment (TedLivist) [inline teacher.rb]: - [ ] Kindly assign `parent_permission` correctly.

  ```
  super(age, name, parent_permission: parent_permission)
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Kgomotso196,

Good job so far!
✔️ Classes are in different files
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @Kgomotso196, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Highlights
- Great job, your project is error free :+1: 
- Nice work you meet all the requirements :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**